riemann-java-client.iml

```diff
@@ -6,23 +6,25 @@
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
```

Comment:
probably not, possibly new intellij version did this when importing the maven project.
otherwise it wouldn't recognize imports etc.